lib/ansible/modules/debconf.py

```diff
@@ -140,14 +140,15 @@ def get_password_value(module, pkg, question, vtype):
     for line in out.split("\n"):
         if line.startswith(pkg):
             desired_line = line
-            break
+            (dpkg, dquestion, dvtype, *dvalue) = line.split()
+            if dquestion == question and dvtype == vtype:
+                if len(dvalue) >= 1:
+                    return dvalue[0]
+                return ''
 
```

Comment:
you are changing the logic for not found, since desired_line will always be true when leaving the loop, the error handler will be wrong